VultisigApp/VultisigApp/Services/SwapService/SwapService.swift

```diff
@@ -59,13 +59,13 @@ struct SwapService {
                 chain: try KyberSwapService.shared.getChainName(for: fromCoin.chain),
                 amount: amount,
                 fromCoin: fromCoin,
-                toCoin: toCoin, isAffiliate: isAffiliate
+                toCoin: toCoin, isAffiliate: false  // KyberSwap uses its own referrer system
```

Comment:
I don't think you can change it here , because `isAffiliate` = `false`, then we will not apply affiliate fee